go-update/src/test/java/org/reactome/release/goupdate/DuplicateReporterIT.java

```diff
@@ -90,19 +97,25 @@ public void testGetDuplicateReferrers() throws Exception
 							{
 								for (GKInstance ref : refs)
 								{
-									System.out.println("\t\t"+ref.toString());
+									if (!Arrays.asList(ReactomeJavaConstants.GO_BiologicalProcess, ReactomeJavaConstants.GO_MolecularFunction, ReactomeJavaConstants.GO_CellularComponent, ReactomeJavaConstants.Compartment).contains(ref.getSchemClass().getName()))
+									{
+										System.out.println("\t\tvia "+refAtt.getName()+":\t"+ref.toString());
```

Comment:
This looks like the end to a statement -- is it a possible continuation from line 85?

Reply:
You mean the print statement? Yes, line 85 prints a "header", and then inside the loop, everything for that header gets printed.